rust/private/rustdoc_test.bzl

```diff
@@ -66,29 +66,34 @@ def _build_rustdoc_test_script(toolchain, dep_info, crate):
     link_flags = []
     link_search_flags = []
 
-    # TODO: This should be able to use ctx.actions.Args()
-    link_flags += ["--extern " + crate.name + "=" + crate.output.short_path]
-    link_flags += ["--extern " + c.name + "=" + c.output.short_path for c in d.direct_crates.to_list()]
+    link_flags += ["--extern=" + crate.name + "=" + crate.output.short_path]
+    link_flags += ["--extern=" + c.name + "=" + c.output.short_path for c in d.direct_crates.to_list()]
     link_search_flags += ["-Ldependency={}".format(dirname(c.output.short_path)) for c in d.transitive_crates.to_list()]
 
     link_flags += ["-ldylib=" + get_lib_name(lib) for lib in d.transitive_dylibs.to_list()]
     link_search_flags += ["-Lnative={}".format(dirname(lib.short_path)) for lib in d.transitive_dylibs.to_list()]
     link_flags += ["-lstatic=" + get_lib_name(lib) for lib in d.transitive_staticlibs.to_list()]
     link_search_flags += ["-Lnative={}".format(dirname(lib.short_path)) for lib in d.transitive_staticlibs.to_list()]
 
+    edition_flags = ["--edition={}".format(crate.edition)] if crate.edition != "2015" else []
```

Comment:
Really minor nit: Can this be converted to use the add_addition_flag fn?

Reply:
I couldn't think of a way to get the fn to work on both 'args' and 'list

i'll take a look at the rustdoc_test todo to use args

Reply:
filed https://github.com/bazelbuild/bazel/issues/6888 for args issue